src/components/SignupConfirmation.tsx

```diff
@@ -0,0 +1,161 @@
+import React, {useState} from 'react';
+import {Button, Checkbox, Form, Input, Radio} from 'antd';
+
+const SignupConfirmation: React.FC = () => {
+    const onFinish = (values: any) => {
+        // send data to redux
+    };
+
+    const inputStyle = {
```

Comment:
We should have all our CSS in less files, no css in js!

Reply:
As in, we want to make a new less file for this component and put all the css in there. Then import and use the classes as needed.